CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 ## it, and/or modify it under the terms of the GNU Lesser General
 ## Public License as published by the Free Software Foundation; either
 ## version 2.1 of the License, or (at your option) any later version.
-## The full text of the license can be found in the file LICENSE at
+## The full text of the license can be found in the file LICENSE.md at
 ## the top level of the deal.II distribution.
```

Comment:
While we are at it, shall we also rename "deal.II distribution" to something like "deal.II repository"?

Reply:
Maybe "the top level directory of deal.II"?

Reply:
I like that. Let me do that change.